src/phasorpy/cluster.py

```diff
@@ -0,0 +1,263 @@
+"""Cluster phasor coordinates.
+
+The `phasorpy.cluster` module provides functions to:
+
+- cluster phasor coordinates using Gaussian Mixture Model (GMM):
+    - :py:func:`phasor_cluster_gmm`
+
+"""
+
+from __future__ import annotations
+
+__all__ = ['phasor_cluster_gmm']
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ._typing import Any, ArrayLike
+
+import numbers
+from collections.abc import Sequence
+
+import numpy
+from sklearn.mixture import GaussianMixture
+
+
+def phasor_cluster_gmm(
+    real: ArrayLike,
+    imag: ArrayLike,
+    scaling: float = 2 * numpy.sqrt(2),
```

Comment:
Maybe its better to use `sigma=2.0` instead of `scaling` as a parameter. Sigma is already used in other functions.